devtools/common/src/main/filtered/extensions.json

```diff
@@ -8,6 +8,16 @@
     "groupId": "io.quarkus",
     "artifactId": "quarkus-agroal"
   },
+  {
+    "name": "ap4k",
+    "labels": [
+      "ap4k",
+      "kubernetes",
+      "openshift"
+    ],
+    "groupId": "io.quarkus",
+    "artifactId": "quarkus-ap4k-deployment"
```

Comment:
This needs to be changed to `quarkus-ap4k`.

Reply:
Oops! Fixed :)